src/Contract/RestInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Contract;
+
+use Psr\Http\Message\ResponseInterface;
+
+interface RestInterface
+{
+    /**
+     * @param array<string, mixed> $headers
+     */
+    public function get(string $uri, array $headers = []): ResponseInterface;
+
+    /**
+     * @param array<string, mixed> $headers
+     */
+    public function postJson(string $uri, string $jsonBody, array $headers = []): ResponseInterface;
+
+    /**
+     * @param array<string, mixed> $headers
+     * @param array<string, string> $formFields
+     */
```

Comment:
the method comments dont match the signatures

Reply:
I did not add all parameters, as only arrays need to be described in more detailed way (it is done like this in all other files)

However, I will change the order of `$headers` and `$formFields`.